h/migrations/versions/b32d09564cf5_correct_timestamps_of_elife_annotations.py

```diff
@@ -0,0 +1,6394 @@
+        "created": "2014-04-02T20:50:17Z",
+        "modified": "2014-04-02T20:50:17Z"
+    },
+    {
+        "imported_id": "disqus-import:2532560110",
+        "created": "2014-04-03T09:30:04Z",
+        "modified": "2014-04-03T09:30:04Z"
+    },
+    {
+        "imported_id": "disqus-import:2532560145",
+        "created": "2014-04-03T09:41:25Z",
+        "modified": "2014-04-03T09:41:25Z"
+    },
+    {
+        "imported_id": "disqus-import:2532560119",
+        "created": "2014-04-03T13:43:09Z",
+        "modified": "2014-04-03T13:43:09Z"
+    },
+    {
+        "imported_id": "disqus-import:2532560123",
+        "created": "2014-04-04T21:52:14Z",
+        "modified": "2014-04-04T21:52:14Z"
+    },
+    {
+        "imported_id": "disqus-import:2532560121",
+        "created": "2014-04-04T22:55:48Z",
+        "modified": "2014-04-04T22:55:48Z"
+    },
+    {
+        "imported_id": "disqus-import:2532560148",
+        "created": "2014-04-06T05:22:33Z",
+        "modified": "2014-04-06T05:22:33Z"
+    },
+    {
+        "imported_id": "disqus-import:2532560157",
+        "created": "2014-04-08T00:01:03Z",
+        "modified": "2014-04-08T00:01:03Z"
+    },
+    {
+        "imported_id": "disqus-import:2532560154",
+        "created": "2014-04-08T06:31:58Z",
+        "modified": "2014-04-08T06:31:58Z"
+    },
+    {
+        "imported_id": "disqus-import:2532560124",
+        "created": "2014-04-08T22:23:12Z",
+        "modified": "2014-04-08T22:23:12Z"
+    },
+    {
+        "imported_id": "disqus-import:2532560161",
+        "created": "2014-04-09T04:44:55Z",
+        "modified": "2014-04-09T04:44:55Z"
+    },
+    {
+        "imported_id": "disqus-import:2532560142",
+        "created": "2014-04-09T18:23:42Z",
+        "modified": "2014-04-09T18:23:42Z"
+    },
+    {
+        "imported_id": "disqus-import:2532560152",
+        "created": "2014-04-10T13:37:05Z",
+        "modified": "2014-04-10T13:37:05Z"
+    },
+    {
+        "imported_id": "disqus-import:2532560115",
+        "created": "2014-04-11T03:08:12Z",
+        "modified": "2014-04-11T03:08:12Z"
+    },
+    {
+        "imported_id": "disqus-import:2532560126",
+        "created": "2014-04-11T20:41:11Z",
+        "modified": "2014-04-11T20:41:11Z"
+    },
+    {
+        "imported_id": "disqus-import:2532560150",
+        "created": "2014-04-23T07:14:56Z",
+        "modified": "2014-04-23T07:14:56Z"
+    },
+    {
+        "imported_id": "disqus-import:2532560162",
+        "created": "2014-05-08T19:30:37Z",
+        "modified": "2014-05-08T19:30:37Z"
+    },
+    {
+        "imported_id": "disqus-import:2532560160",
+        "created": "2014-05-08T20:00:00Z",
+        "modified": "2014-05-08T20:00:00Z"
+    },
+    {
+        "imported_id": "disqus-import:2532560173",
+        "created": "2014-05-09T20:37:52Z",
+        "modified": "2014-05-09T20:37:52Z"
+    },
+    {
+        "imported_id": "disqus-import:2532560183",
+        "created": "2014-05-09T20:38:31Z",
+       
```

Comment:
Could we take this context about the total number of timestamps and put it into this log message? We have access to `len(TIMESTAMPS)`, after all.

Reply:
Sure, that'll be simple